.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Purpose or Intent
+-----------------
+> Describe the rational and use case for this pull request.  Provide any background, examples, and images that provide further information to the accurately describe what it is that you are adding to the repo.  Add subsections as necessary to organize and Feel free to link and reference other PRs as necessary, but also include them in the links section below as a quick reference.
```

Comment:
typo: `Feel free` should be lowercased to `feel free` since it is not starting a sentence